Tests/SessionTests.swift

```diff
@@ -0,0 +1,148 @@
+import Embassy
+@testable import Turbo
+import WebKit
+import XCTest
+
+private let defaultTimeout: TimeInterval = 10000
+private let turboTimeout: TimeInterval = 30
+
+class SessionTests: XCTestCase {
+    private static let eventLoop = try! SelectorEventLoop(selector: try! KqueueSelector())
+    private static let server = DefaultHTTPServer.turboServer(eventLoop: eventLoop)
+
+    private let sessionDelegate = TestSessionDelegate()
+    private var session: Session!
+
+    override class func setUp() {
+        super.setUp()
+        try! server.start()
+        DispatchQueue.global().async { eventLoop.runForever() }
+    }
```

Comment:
```suggestion
    override class func setUp() async throws {
        super.setUp()
        try server.start()
        DispatchQueue.global().async { eventLoop.runForever() }
    }
```

Reply:
Oddly enough there isn't a class-level `setUp()` function that throws. So I think we need to keep this `try!`, sadly.

Reply:
Actually, I can create the server for every test run!